tolk/json_rpc.py

```diff
@@ -20,6 +20,35 @@ def read_coils(self, starting_address, quantity, slave_id=1):
         :param starting_address: Number of starting address.
         :param quantity: Number of coils to read.
         :param slave_id: Number with Slave id, default 1.
+        :returns: JSON-RPC response containing the status of coils on success JSON-RPC error on failure.
+
+        **Example request:**
+
+        .. sourcecode:: json
+
+            {
+                "params":{
```

Comment:
The examples have been parsed by an [online JSON parser](https://jsonformatter.curiousconcept.com/) following the RFC standard and outputs it without spaces between object keys and values. So I think I follow the standard and keep it like this.